HW1/P2/P2b.py

```diff
@@ -1,3 +1,26 @@
 from P2 import *
 
-# Your code here
+import pyspark
+
+from P2 import *
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+sc =pyspark.SparkContext()
+
+data=[]
+for i in range(0,1999):
+    for j in range(0,1999):
+      data.append([i,j])
+
+#print data
+rdd=sc.parallelize(data)
+rdd=rdd.repartition(100)
```

Comment:
Good job by using "rdd.repartition()". I haven't thought of this idea and learn another good way to balance the work from you. Here is my solution for your reference: using random.shuffle() after setting up x and y axis RDD.